docs/apidoc.md

```diff
@@ -0,0 +1,16 @@
+# juCi++ API doc
+
+## Prerequisites:
+ * doxygen
+ * plantuml (install via apt-get or download from http://de.plantuml.com/; if downloaded either copy the jar file to /usr/bin or set the environment variable PLANTUML_PATH to point to the path containing the jar file)
+
```

Comment:
Yes, java is also a dependency to run plantuml. Plantuml allows to specify uml diagrams within doxygen comments.
Without plantuml, the API doc will omit the UML diagrams.